service.go

```diff
@@ -177,8 +178,9 @@ func ConfigureMetrics(ctx context.Context) error {
 
 	log.Info("Created Service")
 
+	path := "/metrics"
```

Comment:
This should be configurable. The operator-sdk generated a '/metrics' endpoint by default so this by default will conflict with that.